docker-compose.yml

```diff
@@ -18,15 +19,42 @@ services:
       - ./cert:/var/eets/test/proxy-conf/cert
     depends_on:
       - "eets-testcontainer-provider-interface-service"
-  eets-testcontainer-messagebroker:
-    image: rabbitmq:3.7.6-management
-    container_name: eets-testcontainer-rabbitmq
-    user: rabbitmq:rabbitmq
+
+  eets-testcontainer-zookeeper:
+    container_name: eets-testcontainer-zookeeper
+    image: bitnami/zookeeper:3.8
+    ports:
+      - "2181:2181"
+    volumes:
+      - "zookeeper-data:/bitnami/zookeeper"
+    environment:
+      - ALLOW_ANONYMOUS_LOGIN=yes
+
+  kafka:
+    image: bitnami/kafka:3.3.1
+    container_name: kafka
+    ports:
+      - "9092:9092"
```

Comment:
Is there a particular reason to bind this port to the host?
In my opinon we should only expose minimal ports to the host as there we don't know where potential users will run this setup